packages/android_intent/test/android_intent_test.dart

```diff
@@ -140,6 +140,52 @@ void main() {
         verifyZeroInteractions(mockChannel);
       });
     });
+
+    group('showChooser', () {
+      test('raises error if the intent has no action', () async {
+        try {
+          androidIntent = AndroidIntent.private(
+            package: 'packageName',
+            componentName: 'componentName',
+            channel: mockChannel,
+            platform: FakePlatform(operatingSystem: 'android'),
+          );
+          await androidIntent.showChooser();
+          fail('should raise a StateError');
+        } on StateError catch (e) {
+          expect(e.message, 'action must not be null or empty string');
+        } catch (e) {
+          fail('should raise a StateError');
+        }
+      });
+
+      test('pass right params', () async {
+        androidIntent = AndroidIntent.private(
+            action: 'action_view',
+            data: Uri.encodeFull('https://flutter.io'),
+            flags: <int>[Flag.FLAG_ACTIVITY_NEW_TASK],
+            channel: mockChannel,
+            platform: FakePlatform(operatingSystem: 'android'),
+            type: 'video/*');
+        await androidIntent.showChooser();
+        verify(mockChannel.invokeMethod<void>('showChooser', <String, Object>{
+          'action': 'action_view',
+          'data': Uri.encodeFull('https://flutter.io'),
+          'flags':
+              androidIntent.convertFlags(<int>[Flag.FLAG_ACTIVITY_NEW_TASK]),
+          'type': 'video/*',
+        }));
+      });
+
+      test('call in ios platform', () async {
```

Comment:
Rather than naming a test by what it does, consider naming the test based on what you're verifying. For example, this test might be called `'does nothing when called on iOS'`.

Reply:
Just used the same name as in the existing test. No problem, I will fix it.